metaquot/ppxlib_metaquot.ml

```diff
@@ -102,11 +154,73 @@ module Expr = Make (struct
 
   let annotate e core_type = pexp_constraint ~loc:core_type.ptyp_loc e core_type
 
-  let cast ext =
+  let fresh_name =
+    let counter = ref 0 in
+    fun () ->
+      let var = "_ppx_metaquot_helper_var%d" ^ string_of_int !counter in
```

Comment:
There is already a `gen_symbol` function available from ppxlib. Let's not mint a new one here.

Reply:
Updated, thanks.